site/content/integrate/slash-commands/_index.md

```diff
@@ -183,3 +183,13 @@ Ensure you are emitting the `Content-Type: application/json` header, otherwise y
 #### Are slash commands Slack-compatible?
 
 See the [admin guide's notes on Slack compatibillity](https://docs.mattermost.com/developer/slash-commands.html#slack-compatibility).
+
+#### How do I use Bot Accounts to reply to slash commands?
+
+##### If you are developing a integration
+- Set up a [Personal Access Token](https://docs.mattermost.com/developer/personal-access-tokens.html) for the Bot Account you want to reply with.
```

Comment:
This section might change with 5.12 release (as there will be a different flow for creating personal access token for bot), but this is good for now.